meas_std_val: 0.05
u_std: [0.05, 0.05, 0.05, 0.05]
initial_range: [1.0, 1.0, 1.0, 0.25, 0.25, 0.25]
num_particles: 250
stale_filter_time: 1.0

labels:
- power_cell

bounds:
  - [-20.0, 20.0]
  - [-20.0, 20.0]
  - [-0.117, 1.0]
  - [-10.0, 10.0]
  - [-10.0, 10.0]
  - [-10.0, 10.0]
